### ISSUE TITLE
bug: Context menu items not visible

### ISSUE THREAD

**Issue by kdpuvvadi** [HO-COOH/FastCopy#4]
Items in submenu of the `FastCopy` are not visible. 

## screenshot

![Menu](https://github.com/HO-COOH/FastCopy/assets/7326255/c9f9c2e7-8eaa-4293-adc2-1acc39c652de)

With info from the screenshots, when click on area for copying,  it say copying successful but stuck on copying

![Copying](https://github.com/HO-COOH/FastCopy/assets/7326255/b4336203-4666-4c7d-9cea-aee3dbe23532)

![Success message](https://github.com/HO-COOH/FastCopy/assets/7326255/5b3a94b0-cb36-44bc-acdb-133535755fc4)

## System info

`Edition`:	       Windows 10 Enterprise
`Version`	       22H2
`OS build:`       19045.2965
`Experience:`    Windows Feature Experience Pack 1000.19041.1000.0

**Comment by HO-COOH** [comment]
Can you please provide your file path, both the original and destination (I guess it's some kind of encoding issues)? Also try to enable `UTF-8 beta support` in windows system language settings and see whether it solves it.

**Comment by kdpuvvadi** [comment]
Source: `C:\data\el\tool\repo\rhome\bom\items.xlsx`
Dest: `C:\Users\kd\Desktop\`

Tried with `UTF-8 beta support` enabled but still the same.

**Comment by HO-COOH** [comment]
> Source: `C:\data\el\tool\repo\rhome\bom\items.xlsx` Dest: `C:\Users\kd\Desktop\`
> 
> Tried with `UTF-8 beta support` enabled but still the same.

Largely a permission issue. Try using command `robocopy "C:\data\el\tool\repo\rhome\bom\" "C:\Users\kd\Desktop\" "items.xlsx"` and see the output. At this time, this project does not handle any errors and will always show success when robocopy exited

**Comment by kdpuvvadi** [comment]
Here's the output 

```
$ robocopy 'C:\data\el\tool\repo\rhome\bom\item.xlsx' 'C:\Users\kd\Desktop\'

-------------------------------------------------------------------------------
   ROBOCOPY     ::     Robust File Copy for Windows
-------------------------------------------------------------------------------

  Started : 24 May 2023 10:27:02
   Source : C:\data\el\tool\repo\rhome\bom\item.xlsx
     Dest : C:\Users\kd\Desktop\

    Files : *.*

  Options : *.* /DCOPY:DA /COPY:DAT /R:1000000 /W:30

------------------------------------------------------------------------------

2023/05/24 10:27:02 ERROR 123 (0x0000007B) Accessing Source Directory C:\data\el\tool\repo\rhome\bom\item.xlsx The filename, directory name, or volume label syntax is incorrect.
```

Seems like with the files it's failing but with the directories, it's working fine. FileCopy's behaviour is the same.

```
 $ robocopy  'C:\tools\eda\' 'C:\Users\Sri Sarada\Desktop\rr\'

-------------------------------------------------------------------------------
   ROBOCOPY     ::     Robust File Copy for Windows
-------------------------------------------------------------------------------

  Started : 24 May 2023 10:31:03
   Source : C:\tools\eda\
     Dest : C:\Users\kd\Desktop\rr\

    Files : *.*

  Options : *.* /DCOPY:DA /COPY:DAT /R:1000000 /W:30

------------------------------------------------------------------------------

                           4    C:\tools\eda\
100%        New File                 140        lin64.sh
100%        New File                 140        mac64.sh
100%        New File                3578        Readme.txt
100%        New File                 113        win64.bat

------------------------------------------------------------------------------

               Total    Copied   Skipped  Mismatch    FAILED    Extras
    Dirs :         1         0         1         0         0         0
   Files :         4         4         0         0         0         0
   Bytes :     3.8 k     3.8 k         0         0         0         0
   Times :   0:00:00   0:00:00                       0:00:00   0:00:00


   Speed :              441222 Bytes/sec.
   Speed :              25.246 MegaBytes/min.
   Ended : 24 May 2023 10:31:03
```

**Comment by HO-COOH** [comment]
@kdpuvvadi One fact I learned the hard way is that, you cannot use `\` in those paths, you need to convert those to `/`, see this [commit here](https://github.com/HO-COOH/FastCopy/commit/d12a8ad68984acd79838ecc8a75c3b8adae3499c#diff-8b82cea1c7af5e0cb5cb9f77f3f3a77d608279ebf386bf4b8b83fd79dce3c9c6R194). Try again, if it succeeded, you can expect this fix in the next release

**Comment by kdpuvvadi** [comment]
it's same with forward or backward slash. one more thing i've observed is, if the directory has space in the name, it outright crashes without any error.

**Comment by HO-COOH** [comment]
You need to put file name after destination if you are copying individual files using `robocopy`. Check its doc

**Comment by franco6996** [comment]
I have the same bugs. 

-  The text not shows on the context menu. 

- After copying some folder i have to close manually the progress window (the files are copied ok). 

![image](https://github.com/HO-COOH/FastCopy/assets/108809411/e3ab9e9d-07fc-4a2a-a2b8-d5eb7cca81ef)

**Comment by Mojas84** [comment]
Hi on Windows 11 too it same bug.....    But app is very good and realy faster please fix it. 🤔

**Comment by Mojas84** [comment]
And next bug is i can owerwrite folder on destination and program crash not reponding

**Comment by HO-COOH** [comment]
okay, I will get a windows 10 VM to investigate.

**Comment by HO-COOH** [comment]
Try the new release @kdpuvvadi

**Comment by Mojas84** [comment]
Hi good but with startallback still copy setting mini menu not visible full. In the copy or move 0procent still to the end.  notification not started

**Comment by Mojas84** [comment]
Graph not visible.

**Comment by Mojas84** [comment]
Please add fuction custom wav sound on finish.

**Comment by Mojas84** [comment]
On network drive copy or move not launched....

**Comment by Mojas84** [comment]
Please add fuction recolored app.....

**Comment by HO-COOH** [comment]
@Mojas84 For issues unrelated to this one, please open another issue. This issue is for context menu on windows 10.

**Comment by Mojas84** [comment]
Ok

**Comment by therealadrianmmiller** [comment]
No context menu options here, using latest release, uninstalled, reinstalled, blank context menu.....

**Comment by therealadrianmmiller** [comment]
> [@therealadrianmmiller](https://github.com/therealadrianmmiller) Does it bahave the same for the classic menu (that you click on the `Show more options`) ? And what's your system language? I have checked my code thoroughly and cannot figure out a reason. I should fallback with English if there is no localized string. Unless my ` SHDupStr` call failed (which is very unlikely)


Yes i toggled classic and  modern context menus as i thought of that too, no change

My system language is English (Australian), with English (United States) still installed and as second preference
